erpnext/crm/doctype/lead/lead.py

```diff
@@ -230,84 +195,60 @@ def set_lead_for_customer(lead, customer):
 @frappe.whitelist()
 def make_opportunity(source_name, target_doc=None):
 	def set_missing_values(source, target):
-		_set_missing_values(source, target)
+		target.opportunity_from = 'Lead'
+		target.run_method('set_missing_values')
 
-	target_doc = get_mapped_doc("Lead", source_name,
-		{"Lead": {
+	target_doc = get_mapped_doc("Lead", source_name, {
+		"Lead": {
 			"doctype": "Opportunity",
 			"field_map": {
-				"campaign_name": "campaign",
-				"doctype": "opportunity_from",
 				"name": "party_name",
-				"lead_name": "contact_display",
-				"company_name": "customer_name",
-				"email_id": "contact_email",
-				"mobile_no": "contact_mobile"
 			}
-		}}, target_doc, set_missing_values)
+		}
+	}, target_doc, set_missing_values)
 
 	return target_doc
 
 
 @frappe.whitelist()
 def make_quotation(source_name, target_doc=None):
 	def set_missing_values(source, target):
-		_set_missing_values(source, target)
+		target.quotation_to = "Lead"
+		target.run_method("set_missing_values")
+		target.run_method("set_other_charges")
```

Comment:
Need to decide between using `set_other_charges` or `append_taxes_from_master`

- set_taxes
- set_other_charges
- append_taxes_from_master